packages/quantic/cypress/e2e/default-1/notifications/notifications.cypress.ts

```diff
@@ -7,6 +7,7 @@ import {
 import {NotificationsExpectations as Expect} from './notifications-expectations';
 
 const exampleNotifications = ['Notification one', 'Notification two'];
+const useCases = ['search', 'insights'];
```

Comment:
No need for this object, we are have another one `useCaseParamTest` used across all the other e2e test, please follow the same pattern.

Reply:
You should have a pattern like the following: 
```
useCaseParamTest.forEach((param) => {
    describe(param.label, () => {
      ...
  })})
````